src/lib/loaders/loaders_with_authentication/convection.ts

```diff
@@ -89,5 +89,10 @@ export default (accessToken, opts) => {
       {},
       { method: "PUT" }
     ),
+    createConsignmentInquiryLoader: convectionLoader(
```

Comment:
#superminor Can we place this alphabetically in the loader list?

Reply:
Noted. I will update it.